index.js

```diff
@@ -1,2 +1,14 @@
+let num1 = 2;
+let num2 = 31;
+let multiply = 62;
 
-// Write your code here
\ No newline at end of file
+let random = 5;
+Math.floor(Math.random() * 5);
```

Comment:
And here: can you figure out a way to randomly generate a number and store the random number in the variable? (You're very close to doing it already).